health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/LatLongMap/LatLongMap.js

```
@@ -0,0 +1,412 @@
+          border: "1px solid #D6D5D4",
+          borderRadius: "4px",
+          maxHeight: "480px",
+          minWidth: "40%",
+          backgroundColor: "#FAFAFA",
+          padding: "40px",
+        }}
+      >
+        <table style={{ width: "100%" }}>
+          <tbody>
+            {Object.keys(tableData[pointName]).map((key) => (
+              <tr style={{ borderBottom: "1px solid #D6D5D4", textAlign: "left" }}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)